defs/scenarios/mysql/mysql_router.yaml

```diff
@@ -0,0 +1,26 @@
+checks:
+  using_mysql_router:
+    apt: mysql-router
+  client_ssl_mode:
+    config:
+      handler: hotsos.core.plugins.mysql.MySQLRouterConfig
+      assertions:
+        key: client_ssl_mode
+        ops: [[eq, PREFERRED]]
+  client_ssl_cert:
+    config:
+      handler: hotsos.core.plugins.mysql.MySQLRouterConfig
+      assertions:
+        key: client_ssl_cert
+        ops: [[ne, None]]
+conclusions:
+  client_ssl_cert_problem:
+    decision:
+      and: [ using_mysql_router, client_ssl_mode ]
+      not: client_ssl_cert
+    raises:
+      type: MySQLWarning
+      message: >-
+          MySQL Router has client_ssl_mode configuration but no client_ssl_cert
```

Comment:
I'd like propose we work like this:

This host is running MySQL Router and has client_ssl_mode configured but
client_ssl_cert is not set. This will cause mysql-router to error when
restarted.

Reply:
Also is there a launchpad bug for this issue? If there is we should add it here as well.

Reply:
done